tibanna/_version.py

```diff
@@ -21,3 +22,7 @@ def _get_project_meta():
 # they can't parse the pyproject.toml. For all other use cases, fall back to the pyproject.toml version
 __version__ = os.environ.get('TIBANNA_VERSION', False) or str(pkg_meta['version'])
 
+# AWSF image version - will default to the minor version of the deployed Tibanna
+tibanna_version = semantic_version.Version(__version__)
+__awsf_image_version__ = str(semantic_version.Version(major=tibanna_version.major, minor=tibanna_version.minor, patch=0))
```

Comment:
I would import `__awsf_image_version__` into `tibanna_ff` as we do with `__version__` to bring that change to CGAP.